client/my-sites/plugins/plan-setup/index.jsx

```diff
@@ -1,81 +1,336 @@
 /**
```

Comment:
@johnHackworth this file needs some work on the namespace and classes (`plan-setup` is too generic) that don't match the folder namespace.